src/cpp/ext/otel/otel_plugin.h

```diff
@@ -395,6 +424,34 @@ class OpenTelemetryPlugin : public grpc_core::StatsPlugin {
     return plugin_options_;
   }
 
+  template <typename ValueType>
+  struct CallbackGaugeState {
+    // It's possible to set values for multiple sets of labels at the same time
+    // in a single callback. Key is a vector of label values and enabled
```

Comment:
Thinking about it if we just do "enabled optional label values", we would run into a case where -
Let's say, we have an Instrument, which has three optional label keys in its descriptor -
{key_a, key_b, key_c} and only `key_a` and `key_b` are enabled in the plugin.
The callback reports two values with labels - {"foo", "bar", "baz"} and {"foo", "bar", "foobar"}.
We would end up overwriting the second value over the first which sounds fine to me.